charts/rd-caseworker-ref-api/values.preview.template.yaml

```diff
@@ -28,6 +28,6 @@ java:
         - name: caseworker-topic-primary-send-listen-shared-access-key
           alias: CWRD_MQ_PASSWORD
         - name: caseworker-ref-api-s2s-secret
-          alias: S2S_SECRET
+          alias: SRD_API_S2S_SECRET
```

Comment:
caseworker-ref-api-s2s-secret needs to be made in capital with underscore  also  for LD-SDK-KEY rename alias to LD-SDK-KEY from RD_LD_SDK_KEY

Reply:
please ignore above .. never mind